pkg/cri/sbserver/cni_conf_syncer.go

```diff
@@ -78,6 +79,17 @@ func (syncer *cniNetConfSyncer) syncLoop() error {
 				logrus.Debugf("cni watcher channel is closed")
 				return nil
 			}
+			if event.Op&fsnotify.Remove > 0 {
+				for {
+					err := syncer.watcher.Add(syncer.confDir)
```

Comment:
need to check the event.Name with confDir if there is just normal file deletion.